app/src/main/java/ch/epfl/sweng/eventmanager/repository/data/Zone.java

```diff
@@ -4,15 +4,28 @@
 import com.google.android.gms.maps.model.PolygonOptions;
 import java.util.List;
 
+/**
+ * This class represents a delineated geographical zone that is overlayed by a blue color using the
+ * addPolygon method.
+ *
+ * This zone is delineated using a list of geographical positions, that can be accessed using
+ * the getPositions method.
+ */
 public class Zone {
     private List<Position> positions;
 
+    /**
+     * constructor
+     *
+     * @param positions that delineated the zone
+     */
     public Zone(List<Position> positions) {
         this.positions = positions;
     }
 
-    public Zone() {}
```

Comment:
Are you sure it works with firebase ?

Reply:
It doesn't work with firebase, I add it in the next commit